src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -128,6 +140,24 @@ const WorkspaceInitialPage = (props) => {
         },
     ];
 
+    const threeDotsMenuItems = [
+        {
+            icon: Expensicons.Trashcan,
+            text: props.translate('workspace.common.delete'),
+            onSelected: () => setIsDeleteModalOpen(true),
+        },
+        {
+            icon: Expensicons.Hashtag,
+            text: props.translate('workspace.common.goToRoom', {roomName: CONST.REPORT.WORKSPACE_CHAT_ROOMS.ADMINS}),
+            onSelected: () => goToRoom(CONST.REPORT.CHAT_TYPE.POLICY_ADMINS),
+        },
+        {
+            icon: Expensicons.Hashtag,
+            text: props.translate('workspace.common.goToRoom', {roomName: CONST.REPORT.WORKSPACE_CHAT_ROOMS.ANNOUNCE}),
+            onSelected: () => goToRoom(CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE),
+        },
```

Comment:
We missed an edge case here. This showed up even when the data was not loaded. This resulted in a crash. https://github.com/Expensify/App/issues/30381